src/main/java/org/plos/namedentity/service/NamedEntityServiceImpl.java

```diff
@@ -179,6 +179,14 @@ public String generateDisplayname(Individualprofile entity, Random rand) {
     return nedDBSvc.getAlerts(frequency);
   }
 
+  public Integer countGlobalTypes() {
+    return nedDBSvc.countGlobalTypes();
+  }
+
+  public Integer countConsumers() {
+    return nedDBSvc.countConsumers();
+  }
+
```

Comment:
Looks great jono, however, it will be even sweeter if had a few tests to back-up the new ned plumbing.